### ISSUE TITLE
How to deal with tables?

### ISSUE THREAD

**Issue by Yashsethi24** [hpanwar08/detectron2#42]
Firstly, thank you so much @hpanwar08 for your contributions!
I was trying to use Mask_RCNN but I observed that whenever data is a little bit spreaded like in a table, then it fails to record it. There is no bbox in that case.
Consider the following picture as a reference.
![Screenshot 2021-02-24 at 6 06 19 PM](https://user-images.githubusercontent.com/68924343/109002097-c1651d00-76cb-11eb-872a-cf755003b318.png)

Can you help me with this?
Thank you

**Comment by hpanwar08** [comment]
Yes, it seems to a problem with the data. Data might not have a lot of examples of this kind of table.

**Comment by Yashsethi24** [comment]
Can you tell me the solution to this?
Can fine tuning the parameters help?

**Comment by hpanwar08** [comment]
Labeling data that has similar types of table might help. Then training on that data.

**Comment by Yashsethi24** [comment]
How many images do I need to train it with? Can I train your model further? If yes, how?

**Comment by hpanwar08** [comment]
Yes you can train the existing model. Process is same for retraining is same as that of new model, just add the existing model path in the config file.
